contrib/guix/build.sh

```diff
@@ -0,0 +1,37 @@
+set -ex
```

Comment:
Should specify which shell is to be used with shebang line?

Reply:
Not necessarily. You can limit your shell syntax to the least common denominator of posix syntax supported by the most minimal shells (busybox, dash, etc.) Specify a specific other shell if you want convenience of syntax features of that particular shell?

Reply:
@practicalswift Didn't see this... But yes @wtogami is right, I try to conform to the common denominator of POSIX sh for portability's sake.